app/bundles/CoreBundle/Doctrine/Mapping/ClassMetadataBuilder.php

```diff
@@ -243,10 +243,11 @@ public function addPublishDates()
      *
      * @return $this
      */
-    public function addDateAdded($nullable = false)
+    public function addDateAdded($nullable = false, int $precision = 0)
```

Comment:
```suggestion
    public function addDateAdded(bool $nullable = false, int $precision = 0): static
```
nitpick: Minor thing